annex/examples/annexA/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+find_package(Klepsydra)
+
+cmake_minimum_required(VERSION 2.8.11)
+SET(PROJ_NAME kpsr_${PROJ_MAIN_NAME}_annexA)
```

Comment:
Going to the build folder and compiling this does not compile the examples of the annex. You have to add the subfolder to the root CMakeLists.txt.

Right now by compiling this the executables get named as `kpsr__annexA_8` etc. When you add the correct code to the root CMakeLists the files will automatically get named `kpsr_tutorial_annexA_8`

Reply:
Done